src/components/ComponentStatus/ComponentStatus.js

```diff
@@ -77,8 +77,6 @@ class ComponentStatus extends React.Component {
         <td>
           {currentItem.item} {tag}
         </td>
-        <td>{currentItem.added}</td>
```

Comment:
Just wanted to double-check if removal here is intentional.

Reply:
Yes, was removing this column since we are showing vanilla, react, angular and experimental the version things were added didn't seem as relevant.